backend/src/test/java/sonia/meetapp/app/participants/IntegrationTest.java

```diff
@@ -29,4 +31,17 @@ void getAllParticipants() throws Exception {
                         []
                         """));
     }
+
+    @DirtiesContext
+    @Test
+    void addParticipant() throws Exception {
+
+        mockMvc.perform(post("/participants")
+                        .contentType(APPLICATION_JSON)
+                        .content("""
+                                {"name":"Mike"}
+                                 """)
+                )
+                .andExpect(status().is(201));
+    }
```

Comment:
Du solltest prüfen ob auch alle Werte richtig gesetzt sind. In deinem Fall ob der Name stimmt und auch ob eine Id generiert wurde.